src/test/java/nl/tudelft/ti2206/group9/gui/EndToEndTest.java

```diff
@@ -148,6 +139,18 @@ private void goThroughShop() {
 		shopScreen(SHOP_BACK);
```

Comment:
Please extend this part of the EndToEnd test to actually test the shop.

<review,testing